WordPress/src/main/java/org/wordpress/android/GCMIntentService.java

```diff
@@ -76,17 +82,17 @@ protected void onError(Context context, String errorId) {
     }
 
     private void handleDefaultPush(Context context, Bundle extras) {
-        long wpcomUserID = AccountHelper.getDefaultAccount().getUserId();
-        String userIDFromPN = extras.getString("user");
-        // userIDFromPN is always set server side, but better to double check it here.
-        if (userIDFromPN != null) {
-            if (!String.valueOf(wpcomUserID).equals(userIDFromPN)) {
+        long wpcomUserId = AccountHelper.getDefaultAccount().getUserId();
+        String pushUserId = extras.getString(PUSH_ARG_USER);
+        // pushUserId is always set server side, but better to double check it here.
+        if (pushUserId != null) {
```

Comment:
Minor nit: no need for the separate null check since the `equals` comparison below returns false if `pushUserId` is null.